src/Platforms/OraclePlatform.php

```diff
@@ -478,6 +478,12 @@ public function getCreateViewSQL($name, $sql)
      */
     public function getDropViewSQL($name)
     {
+        $id = new Identifier($name);
+
+        if(!$id->isQuoted() && strtoupper($name) !== $name) {
+            $name = $this->quoteIdentifier($name);
```

Comment:
This isn't how quoting is supposed to work. It should be the API consumer that either quotes the identifier thereby stating that the case should be preserved, or the platform converts it to the upper case. Quoting the identifier implicitly would be a BC break.

Reply:
Deriving the originally used identifier name for the passed one is a workaround because Oracle and the Oracle platform driver upper case all identifier names automatically if they are not quoted. But the schema manager gets the identifier names back from the database without quotes and we need to pass them exactly as before to the database again so the objects are found. This was not the case for indexes, sequences and views before and thus, dropping or renaming those objects fails when using the schema diff to generate the necessary SQL commands.

It's also no BC break because these methods are only used by the schema diff object and they fix what failed before. Indexes/sequences/views that have been created with or without quotes before either by DBAL or manually still have the same case as before in the schema object and nothing has changed there.